.github/workflows/ci.yml

```diff
@@ -76,25 +76,10 @@ jobs:
       - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ env.RUST_VERSION }}
-      - name: setup binstall
-        uses: taiki-e/install-action@cargo-binstall
       - name: Install Cargo.toml linter
-        run: cargo binstall --no-confirm cargo-sort
+        run: cargo install cargo-sort
```

Comment:
Do we ensure the version is pinned somewhere? Or what happens when e.g. v3 comes out and it formats things differently?

Reply:
No we don't. We didn't pin before, we don't pin now. We'll just update it again. We can  pin it if you feel like it, but that can be done in a separate PR as well.